.gitignore

```diff
@@ -5,4 +5,8 @@ node_modules/
 npm-debug.log
 coverage/
 
-package-lock.json
\ No newline at end of file
+package-lock.json
```

Comment:
I don’t know about the context in which it was originally added, but I also put it in the `.gitignore` for libraries (as opposed to applications), because it doesn’t make much sense to me to have a list of fixed dependency versions when your library’s consumers aren’t going to get the versions that are specified in it anyway.